.env.sample

```diff
@@ -9,7 +9,7 @@ HARBOR_PASSWORD=''
 HARBOR_BASE_URL='http://127.0.0.1:8083'
 HARBOR_BASE_REPO_URL='harbor.dev.eks.otomi.cloud'
 OIDC_CLIENT_SECRET=''
-OIDC_ENDPOINT='https://keycloak.dev.gke.otomi.cloud'
+OIDC_ENDPOINT='https://keycloak.dev.gke.otomi.cloud/realms/otomi'
```

Comment:
I am confused, I think we already do a string replace in our gotmpl logic wrt `/realms/otomi` (e.g. line 8 harbor.gotmpl)?

Reply:
this is env.sample that intended to be use if you run task without otomi-core